src/action.ts

```diff
@@ -26,21 +26,27 @@ async function downloadUpx(): Promise<string> {
     throw "unsupported OS";
 }
 
-async function runUpx(file: string, upx_path: string) {
-    child_process.execSync(`strip ${file}`);
-    child_process.execSync(`${upx_path} ${file}`);
-}
-
 export async function run() {
     try {
         const file = core.getInput('file', { required: true });
+        const args = core.getInput('args');
+        const strip = core.getInput('strip') || 'true';
+        const strip_args = core.getInput('strip_args');
 
         if (!fs.existsSync(file)) {
-            core.setFailed(`File ${file} wasn't found.`);
+            core.setFailed(`⛔ File ${file} wasn't found.`);
         }
 
+        if (/true/i.test(strip)) {
+            console.log('🏃 Running strip...');
```

Comment:
I think you're supposed to use [info()](https://github.com/actions/toolkit/blob/master/packages/core/src/core.ts#L138) instead but I'm not sure. Opinions?

Reply:
Yeah maybe we should!